challenge-manager/challenge-tree/paths.go

```diff
@@ -110,22 +109,25 @@ func (ht *RoyalChallengeTree) IsConfirmableEssentialNode(
 	}
 	// An essential node is confirmable if all of its essential paths
 	// down the tree have a path weight >= the confirmation threshold.
-	// To do this, we compute the path weight of each path and insert
-	// into a min heap. Then, it is sufficient to check that the minimum
-	// element of the heap is >= the confirmation threshold.
-	pathWeights := newPathWeightMinHeap()
+	// To do this, we compute the path weight of each path and find the minimum.
+	// Then, it is sufficient to check that the minimum is >= the confirmation threshold.
```

Comment:
I would like to find a linter that would wrap doc comments at 80 columns. But, I haven't found it yet. Would you mind rewrapping this?